RHDEVDOCS-6275: Adding missing properties in Argo CD CR and Repo server properties tables for GitOps 1.15 (#85114)
openshift/openshift-docs: modules/gitops-argo-cd-properties.adoc

  * `scmProviders` - The URLs of the allowed Source Code Manager (SCM) providers.
  * `<scmRootCAConfigMap>` - The name of the config map that stores the Gitlab SCM Provider's TLS certificate that will be mounted on the Application Set Controller at the "/app/tls/scm/cert" path.
  * `<webhookServer>` - Defines the available options for the ApplicationSet webhook server.
  * `annotations` - List of custom annotations to add to pods deployed by the Operator. This field is optional.
  * `labels` - List of custom labels to add to pods deployed by the Operator. This field is optional.

>   * `volumes` - List of addition volumes configured for the Argo CD ApplicationSet Controller component. This field is optional.

Reviewer: ```suggestion
  * `volumes` - List of addition volumes configured for the Argo CD `ApplicationSet` Controller component. This field is optional.
```

>   * `volumeMounts` - List of addition volume mounts configured for the Argo CD ApplicationSet Controller component. This field is optional.

Reviewer: ```suggestion
  * `volumeMounts` - List of addition volume mounts configured for the Argo CD `ApplicationSet` Controller component. This field is optional.
```

Author: Fixed.

  * `<env>` - Environment to set for the application controller workloads.
  * `<sourceNamespaces>` - The list of non-control plane namespaces for creating and managing Argo CD `Application` resources in target namespaces.
  * `<extraCommandArgs>` - List of arguments added to the existing arguments set by the Operator.

>   * `annotations` - The number of custom annotations that are added to pods deployed by the Operator.
>   * `labels` - The number of custom labels that are added to pods deployed by the Operator.

Reviewer 2: ```suggestion
  * `annotations` - List of custom annotations to add to pods deployed by the Operator. This field is optional.
  * `labels` - List of custom labels to add to pods deployed by the Operator. This field is optional.
```
Can we update with this for all labels & annotations properties in this PR?

Author: Fixed.

>   * `volumes` - The number of addition volumes configured for the Argo CD Application Controller component. This field is optional.
>   * `volumeMounts` - The number of addition volume mounts configured for the Argo CD Application Controller component. This field is optional.

Reviewer 2: ```suggestion
  * `volumes` - List of addition volumes configured for the Argo CD Application Controller component. This field is optional.
  * `volumeMounts` - List of addition volume mounts configured for the Argo CD Application Controller component. This field is optional.
```
Can we update with this for all volumes & volumeMounts properties in this PR?

Author: Fixed.

  * `<logFormat>` - The log format used by the Argo CD server component. Valid options are `text` and `json`.
  * `<env>` - Environment to set for the server workloads.
  * `<enabled>` - The flag to enable Argo CD server during the Argo CD installation.
  * `volumes` - List of addition volumes configured for the Argo CD Application Controller component. This field is optional.
  * `volumeMounts` - List of addition volume mounts configured for the Argo CD Application Controller component. This field is optional.
  * `initContainers` - List of `init` containers for the Argo CD Application Controller component. This field is optional.
  * `sidecarContainers` - List of `sidecar` containers for the Argo CD Application Controller component. This field is optional.
  * `annotations` - List of custom annotations to add to pods deployed by the Operator. This field is optional.
  * `labels` - List of custom labels to add to pods deployed by the Operator. This field is optional.

>   * `enableRolloutsUI` -   Enables the Argo Rollouts UI extension in Argo CD when it is set to true. The default value is set to false.

Reviewer: ```suggestion
  * `enableRolloutsUI` -   When the parameter is set to `true`, the parameter enables the Argo Rollouts UI extension in Argo CD. The default value is set to `false`.
```

Author: Fixed.

  * `<logFormat>` - The log format used by the Argo CD server component. Valid options are `text` and `json`.
  * `<env>` - Environment to set for the server workloads.
  * `<enabled>` - The flag to enable Argo CD server during the Argo CD installation.
  * `volumes` - The number of addition volumes configured for the Argo CD Application Controller component. This field is optional.
  * `volumeMounts` - The number of addition volume mounts configured for the Argo CD Application Controller component. This field is optional.
  * `initContainers` - List of `init` containers for the Argo CD Application Controller component. This field is optional.
  * `sidecarContainers` - List of `sidecar` containers for the Argo CD Application Controller component. This field is optional.
  * `annotations` - The number of custom annotations that are added to pods deployed by the Operator.
  * `labels` - The number of custom labels that are added to pods deployed by the Operator.

>   * `enableRolloutsUI` - The extension used for enabling the Argo Rollouts UI in Argo CD when it is set to
>   true.

Reviewer 2: ```suggestion
  * `enableRolloutsUI` - Enables the Argo Rollouts UI extension in Argo CD when it is set to true. Default is false.
```
